tests/Processor/CommandProcessorTest.php

```diff
@@ -176,8 +172,7 @@ private function getProcessBuilderMock($success = true, array $arguments = [])
         }
 
         $builder->expects(static::exactly(2))
-                ->method('setPrefix')
-                ->with(static::logicalOr(['path/to/php', 'path/to/command', 'foo'], []));
+                ->method('setPrefix');
```

Comment:
there is still arguments to `setPrefix` (should be `[PHP_BINARY, 'path/to/console`]` or something like that)